sovtoken/setup.py

```diff
@@ -19,7 +19,7 @@
 with open(os.path.join(here, 'sovtoken', '__metadata__.py'), 'r') as f:
     exec(f.read(), metadata)
 
-tests_require = ['pytest', 'pytest-xdist', 'mock', 'python3-indy==1.8.2']
+tests_require = ['pytest', 'pytest-xdist', 'mock', 'python3-indy==1.8.3-dev-1088']
```

Comment:
Is it OK that we use `libindy=1.8.3~1094` (in Docker), and `python3-indy==1.8.3-dev-1088` for the wrapper?